lib/OpenQA/WebAPI/Plugin/ObsRsync.pm

```diff
@@ -278,11 +278,7 @@ sub _read_test_id {
 
 sub _get_test_result {
     my ($c, $id) = @_;
-    my $job;
-    eval { $job = $c->schema->resultset("Jobs")->single({id => $id}); 1; }
-      or log_error("Error while trying to retrieve job in _get_test_result: $@");
-
-    return 'unknown' unless $job;
+    return 'unknown' unless my $job = $c->schema->resultset("Jobs")->single({id => $id});
```

Comment:
Since this is changed anyways: Why not just `$c->schema->resultset('Jobs')->find($id)`?

Reply:
Fair point, i'll change it.